.mailmap

```diff
@@ -7,7 +7,6 @@ Alex Torrance <alex.torrance@digital.cabinet-office.gov.uk> <hello@alextorrance.
 ambrozic <ambrozic@gmail.com> <ambrozic@users.noreply.github.com>
 Ana Fernandez <ana.fernandez@digital.cabinet-office.gov.uk> <afda16@users.noreply.github.com>
 Andrea Grandi <a.grandi@gmail.com> <andreagrandi@gds5060.local>
-Andrew Garner <andrew.garner@digital.cabinet-office.gov.uk> <andrew@andrewgarner.com>
```

Comment:
Users shouldn't removed from the mailmap, it's used by git to attribute commits where someone has used multiple email addresses

Reply:
cool, sorry. I will revert that. Was trying to solve another error with that user on my branch but someone told me the reason.